client/FxAccountClient.js

```diff
@@ -1031,4 +1099,30 @@ define([
   return FxAccountClient;
 });
 
+var METRICS_CONTEXT_OPTIONS = {
+  context: true,
+  entrypoint: true,
+  migration: true,
+  service: true,
+  utmCampaign: true,
```

Comment:
[yes, the case is converted](https://github.com/mozilla/fxa-auth-server/pull/1213/files#diff-3833ef223ad2c72e2c7db2440f69353eR36).